src/main/java/com/softserve/academy/spaced/repetition/domain/Person.java

```diff
@@ -17,8 +18,17 @@
     @Column(name = "last_name")
     private String lastName;
 
+
+    @Column(name = "IMAGETYPE", length = 50)
+    @NotNull
+    @Enumerated(EnumType.STRING)
+    private ImageType typeImage;
+
     private String image;
 
+    @Column(name = "imagebase64", columnDefinition = "LONGTEXT")
+    private String imageBase64;
```

Comment:
This field should be LAZY loaded. It is too expensive to extract an image always.